uber-core/src/main/java/com/uber/sdk/core/auth/AccessTokenAuthenticator.java

```diff
@@ -90,10 +93,11 @@ public AccessTokenStorage getTokenStorage() {
     }
 
     synchronized Request doRefresh(Response response) throws IOException {
-        if (signedByOldToken(response, tokenStorage.getAccessToken())) {
-            return resign(response, tokenStorage.getAccessToken());
+        final AccessToken token = tokenStorage.getAccessToken();
```

Comment:
Is there a possibility someone calls `refresh` when they are actually logged out?